### DISCUSSION TITLE
Looking to hire a contractor to help our company get SendPortal up and running using SMTP

### DISCUSSION THREAD

**Discussion by arfouquette** [mettle/sendportal#220] [Help]
We are loving the Sendport app but struggling with sending mail.  We would like to hire some help.  Please let me know if anyone would be interested.  Thanks

**Comment by schooldrift-ceo** [comment]
I can be of helpo to you. already configured to some businesses. contact me: favourafula@gmail.com

**Comment by deadlyanony** [comment]
Yes, I can customise this portal to meet your business needs. 
I also set your SMTP and Templates with Subscriber List Tags. 
I also set your Brand Logo on the Portal. 
Contact : pswork44@gmail.com
